tfx/examples/custom_components/container_components/download_grep_print_pipeline_on_kubernetes.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
```

Comment:
Because this is not yet a suggested way to run things, can you move this to `tfx/orchestration/experimental/kubernetes/examples`?